src/stateless/icon.vue

```diff
@@ -4,6 +4,13 @@
             <div class="icon-caret"></div>
         </div>
 
+        <svg v-if="name === 'schedule-clock'" :style="styleOverride" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+            <g fill-rule="nonzero">
+                <path d="M8 0C3.6 0 0 3.6 0 8s3.6 8 8 8 8-3.6 8-8-3.6-8-8-8zm0 14c-3.3 0-6-2.7-6-6s2.7-6 6-6 6 2.7 6 6-2.7 6-6 6z"/>
```

Comment:
add class="path" to the SVG so its properly styled from outside

Reply:
fixed